exdb/tests/browser_tests.py

```diff
@@ -5,20 +5,40 @@
 import tempfile
 import copy
 import json
+from unittest import SkipTest
 
 import selenium
 from selenium import webdriver
 from selenium.webdriver.common.by import By
 
+from django.test import Client
 from django.test.runner import DiscoverRunner
 from django.utils.translation import ugettext as _
+from django.contrib.auth import get_user_model
+from django.contrib.sessions.models import Session
 from django.contrib.staticfiles.testing import StaticLiveServerTestCase
 from django.conf import settings
 from django.core.urlresolvers import reverse
 
 
-class CustomRunner(DiscoverRunner):
+class CustomRunnerMetaClass(DiscoverRunner.__class__):
```

Comment:
For clarity purposes, `DiscoverRunner.__class__` should be changed to `type(DiscoverRunner)`